### ISSUE TITLE
Fine-tuning on nuscenes

### ISSUE THREAD

**Issue by jo1jun** [PRBonn/TARL#3]
Thanks for sharing the paper and code.

The paper showed the results of fine-tuning on nuscenes.

The SegContrast repo isn't able to fine-tuneing on nuscenes dataset.

Is there any code for fine-tuning on nuscenes?

**Comment by nuneslu** [comment]
Hi! For nuscenes we have used this [repo](https://github.com/PRBonn/nuscenes2kitti) to convert nuscenes to the SemanticKITTI format and so we could use the same dataloader from segcontrast with few changes.

**Comment by jo1jun** [comment]
thanks!! :)

**Comment by malicd** [comment]
Hi @nuneslu,
is there any way you could provide `data_map.py` file and training configuration (epochs, batch size, optimizer parameters) for nuScenes mini and full dataset? This would be really helpful.
Thanks!

**Comment by nuneslu** [comment]
Hi, sure! So for nuScenes this is the command to train for nuscenes and the hyperparameters used:

```
python3 downstream_train.py --use-cuda --use-intensity --checkpoint scratch_nuscenes_semseg --percentage-labels 1.0 --batch-size 8 --sparse-model MinkUNet --epochs 15 --lr 0.001 --optimizer AdamW --data-dir ./Datasets/nuscenes --dataset NuScenes
```

For all the pre-trainings we used the same parameters.

And this is the nuscenes data map yaml file we used:

[nuscenes_map.yaml](https://github.com/PRBonn/TARL/files/12484210/nuscenes_map.txt)

**Comment by malicd** [comment]
Thanks for the provided info!

I would have just a few additional questions.
Did you fine-tune a model for 15 epochs for both mini and full nuScenes subset?
In the provided YAML file, `content` dict is missing (ratio of class points with overall number of points). What values did you use here?

**Comment by nuneslu** [comment]
Yes, for the nuScenes experiment, we trained mini and full nuScenes subsets with 15 epochs. The `content` should not be necessary since we use the normal cross-entropy loss without per-class weights, so we don't have it defined.
